tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
same comment here as I had for `cli.sh` - we should have a README or something that explains how to use these files in the context of `storj-sim` or even prod

Reply:
readme updated